dace/codegen/targets/cpu.py

```diff
@@ -1,12 +1,15 @@
 import itertools
 import warnings
 
-from dace import data, registry, memlet as mm
+import dace
+from dace import data, dtypes, registry, memlet, subsets, symbolic
```

Comment:
I don't think this is good to do at all. Within a python package you should import the modules as specifically as possible, so as to avoid import loops. I'm trying to avoid any instance of `import dace` from within dace

Reply:
You are commenting on `import dace`, not on `from dace import`, right?

Reply:
Removed `import dace`